packages/docs/src/Toaster.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import {
+  Toaster,
+  ToasterContent,
+  ToasterTitle,
+  ToasterButton,
+  Text,
+  IconBox,
+  Flex,
+} from '@citric/core'
+import { InfoCircle, Times } from '@citric/icons'
+
+export default {
+  title: 'Components/Toaster',
+  subcomponents: { ToasterContent, ToasterTitle, ToasterButton },
+} as ComponentMeta<typeof Toaster>
```

Comment:
```suggestion
} 
```

Melhor remover essas definições, o código exibido no "show code" do Figma está incompreensível